fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -281,17 +281,20 @@ public enum ErrorCode {
     /**
      * 10000 - 10099: warehouse
      */
-    ERR_UNKNOWN_WAREHOUSE(10001, new byte[] {'4', '2', '0', '0', '0'}, "Warehouse '%s' not exist."),
-    ERR_WAREHOUSE_EXISTS(10002, new byte[] {'4', '2', '0', '0', '0'}, "Warehouse '%s' already exists."),
-    ERR_WAREHOUSE_SUSPENDED(10003, new byte[] {'4', '2', '0', '0', '0'}, "Warehouse '%s' has been suspended."),
-    ERR_WAREHOUSE_UNAVAILABLE(10004, new byte[] {'4', '2', '0', '0', '0'}, "Warehouse '%s' is not available."),
-    ERR_NO_NODES_IN_WAREHOUSE(10005, new byte[] {'4', '2', '0', '0', '0'},
+    ERR_UNKNOWN_WAREHOUSE(10001, new byte[]{'4', '2', '0', '0', '0'}, "Warehouse '%s' not exist."),
```

Comment:
plz fix the sytle problem, we have a checkstyle rule file under `fe/checkstyle.xml`, you can apply that in intellij

Reply:
thx,I have checked and made the required formatting corrections. The issue may have occurred because a certain plugin automatically optimized the formatting previously